### PR TITLE
Add IPython notebook (and the Python source) to be used for an scheduled Spark job.

### PR DESCRIPTION
**Pull request by jezdez**
Refs #1.

### CODE REVIEW

File: notebooks/aggregated_crashes.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+NOW = datetime.datetime.now()
+
+
+def get_crash_pings(channel, start_date, end_date, fraction):
+    pings = get_pings(
+        sc,  # noqa
+        app='Firefox',
+        channel=channel,
+        submission_date=(start_date, end_date),
+        build_id=('20100101000000', '99999999999999'),
```
Comment (robhudson) [inline notebooks/aggregated_crashes.py]: I think you can leave out the `build_id` if you specify the `submission_date`?

Comment (jezdez) [inline notebooks/aggregated_crashes.py]: Not sure, been following what https://github.com/mfinkle/user-data-analytics/blob/master/android-clients.ipynb does, python_moztelemetry doesn't say anything about that, so I wonder if that's an optimization or just an oversight :-/

**Comment by robhudson** [comment]
Very nicely done. r+wc

File: notebooks/aggregated_crashes.py (hunk 1 above)
Comment (robhudson) [inline notebooks/aggregated_crashes.py]: Ok. let's leave it as-is then. Thanks.

**Pull request merged**